classes/student.rb

```diff
@@ -6,6 +6,11 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students << self unless classroom.students.include?(self)
+  end
+
```

Comment:
Your `student` class should have an `attr_reader: classroom` method to allow you access to the `classroom` class 😉